blockchainetl/jobs/exporters/kafka_exporter.py

```diff
@@ -38,16 +38,24 @@ def get_connection_url(self):
     def open(self):
         pass
 
-    def export_items(self, items):
+    def export_items(self, items):        
         for item in items:
             self.export_item(item)
 
     def export_item(self, item):
         item_type = item.get('type')
-        if item_type is not None and item_type in self.item_type_to_topic_mapping:
+        item_id = item.get('id')
+        
+        if item_id is not None and item_type is not None and item_type in self.item_type_to_topic_mapping:
+            item_type = self.item_type_to_topic_mapping[item_type]
             data = json.dumps(item).encode('utf-8')
-            # logging.debug(data)
-            return self.producer.send(self.item_type_to_topic_mapping[item_type], value=data)
+            
+            if not self.already_processed(item_type, item_id):
+                logging.info(f'Processing message of Type=[{item_type}]; Id=[{item_id}]')
+                self.mark_processed(item_type, item_id)
+                return self.producer.send(item_type, value=data)
```

Comment:
Add error handling, write to logs that. Item has marked as processed but couldn't be produced.